garden_ai/mcp/config_add.py

```diff
@@ -0,0 +1,100 @@
+import os
+import json
+from pathlib import Path
+import subprocess
+import platform
+
+
+class MCPConfigInitalizer:
+    @staticmethod
+    def claude():
+        system = platform.system()
+        if system == "Darwin":  # macOS
+            config_path = (
+                Path.home()
+                / "Library"
+                / "Application Support"
+                / "Claude"
+                / "claude_desktop_config.json"
+            )
+        elif system == "Windows":
+            config_path = (
+                Path(os.environ.get("APPDATA", ""))
+                / "Claude"
+                / "claude_desktop_config.json"
+            )
+        else:  # Linux
+            config_path = (
+                Path.home() / ".config" / "Claude" / "claude_desktop_config.json"
+            )
+
+        return MCPConfigInitalizer._initialize_config_file(config_path)
+
+    @staticmethod
+    def claude_code():
```

Comment:
style nit: since the methods are doing something side-effectful we should give them verb names (`setup_claude_code`) instead of just noun names. Without having read the code I'd probably have guessed that the `claude_code()` method just finds and returns the path to the config, not modify it